packages/adapters/eventstore-adapters/resolve-eventstore-lite/src/init.ts

```diff
@@ -1,34 +1,90 @@
-import getLog from './js/get-log'
-import initEventStore from './js/init'
+import { EventstoreResourceAlreadyExistError } from 'resolve-eventstore-base'
+import { AGGREGATE_ID_SQL_TYPE } from './constants'
+import getLog from './get-log'
 import { AdapterPool } from './types'
-import { AGGREGATE_ID_SQL_TYPE } from './js/constants'
 
-const initSecretsStore = async (pool: AdapterPool): Promise<any> => {
-  const { secretsDatabase, secretsTableName, escapeId } = pool
-  const log = getLog('initSecretsStore')
+const init = async (pool: AdapterPool): Promise<any> => {
+  const log = getLog('init')
+  log.debug('initializing databases')
+  const {
+    database,
+    eventsTableName,
+    snapshotsTableName,
+    secretsTableName,
+    escapeId,
+    config,
+  } = pool
+
+  try {
+    await database.exec(
+      `CREATE TABLE ${escapeId(eventsTableName)}(
+        ${escapeId('threadId')} BIGINT NOT NULL,
+        ${escapeId('threadCounter')} BIGINT NOT NULL,
+        ${escapeId('timestamp')} BIGINT NOT NULL,
+        ${escapeId('aggregateId')} ${AGGREGATE_ID_SQL_TYPE} NOT NULL,
+        ${escapeId('aggregateVersion')} BIGINT NOT NULL,
+        ${escapeId('type')} VARCHAR(700) NOT NULL,
+        ${escapeId('payload')} JSON NULL,
+        PRIMARY KEY(${escapeId('threadId')}, ${escapeId('threadCounter')}),
+        UNIQUE(${escapeId('aggregateId')}, ${escapeId('aggregateVersion')})
+      );
+      CREATE INDEX ${escapeId('aggregateIdAndVersion-idx')} ON ${escapeId(
+        eventsTableName
+      )}(
+        ${escapeId('aggregateId')}, ${escapeId('aggregateVersion')}
+      );
+      CREATE INDEX ${escapeId('aggregateId-idx')} ON ${escapeId(
+        eventsTableName
+      )}(
+        ${escapeId('aggregateId')}
+      );
+      CREATE INDEX ${escapeId('aggregateVersion-idx')} ON ${escapeId(
+        eventsTableName
+      )}(
+        ${escapeId('aggregateVersion')}
+      );
+      CREATE INDEX ${escapeId('type-idx')} ON ${escapeId(eventsTableName)}(
+        ${escapeId('type')}
+      );
+      CREATE INDEX ${escapeId('timestamp-idx')} ON ${escapeId(eventsTableName)}(
+        ${escapeId('timestamp')}
+      )
+      `
+    )
+    log.debug(`events database tables are initialized`)
+
+    await database.exec(
+      `CREATE TABLE ${escapeId(snapshotsTableName)} (
+        ${escapeId('snapshotKey')} TEXT,
+        ${escapeId('content')} TEXT,
+        PRIMARY KEY(${escapeId('snapshotKey')})
+        )`
+    )
```

Comment:
Dependent calls.
If the first call is successful, and the second is unsuccessful, then it is difficult to repeat the creation